tests/templates/test_subroutines.py

```diff
@@ -344,26 +345,23 @@ class TestSingleExcitationUnitary:
                       [16, qml.Hadamard, [0], []], [17, qml.RX, [2], [np.pi / 2]],
                       [4, qml.RZ, [2], [np.pi / 6]], [13, qml.RZ, [2], [-np.pi / 6]]]
              ),
-
```

Comment:
Be careful with all these changes as they are being applied to other parts of the code! I guess it's the result of running black, but it makes code reviewing difficult. I have to scroll down quite a bit to finally see the tests for this template